src/js/nav/globalNav.js

```diff
@@ -132,6 +137,11 @@ export const grouppedNav = {
                         id: 'sources',
                         title: 'Catalog',
                         reload: 'sources'
+                    },
+                    {
+                        id: 'rbac',
+                        title: 'RBAC',
```

Comment:
Title should be "User management"
I can only find this wireframe at the moment: https://redhat.invisionapp.com/share/2XQODDFK4T3#/screens/353063900

Reply:
Fixed.